org.eclipse.dawnsci.remotedataset.core/src/org/eclipse/dawnsci/remotedataset/ServiceHolder.java

```diff
@@ -33,39 +33,39 @@ public static IClassLoaderService getClassLoaderService() {
 		return classLoaderService;
 	}
 
-	public static void setClassLoaderService(IClassLoaderService classLoaderService) {
+	public void setClassLoaderService(IClassLoaderService classLoaderService) {
 		ServiceHolder.classLoaderService = classLoaderService;
 	}
 
 	public static IDownsampleService getDownService() {
 		return downService;
 	}
 
-	public static void setDownService(IDownsampleService downService) {
+	public void setDownService(IDownsampleService downService) {
 		ServiceHolder.downService = downService;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)